<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="15" hostname="NAIM" failures="2" tests="11" name="TutorialNinja functional Negative Test" time="71.813" errors="0" timestamp="2023-10-24T02:29:50 EDT">
  <testcase classname="com.tutorialNinja.functionalTests.R1dot3_TS012_CheckoutFunctionality" name="R1dot3_TS012_TC001" time="10.918"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC002" time="13.115">
    <skipped/>
  </testcase> <!-- r1dot1_TS002_TC002 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC004" time="13.518"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC006" time="9.88"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC002" time="13.201">
    <skipped/>
  </testcase> <!-- r1dot1_TS002_TC002 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC029" time="9.53"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC002" time="9.801">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC002(R1dot1_TS002_LoginFunctionality.java:42)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- r1dot1_TS002_TC002 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC028" time="11.704"/>
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC003" time="9.469">
    <skipped/>
  </testcase> <!-- r1dot1_TS002_TC003 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC003" time="10.468">
    <skipped/>
  </testcase> <!-- r1dot1_TS002_TC003 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC003" time="10.933">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC003(R1dot1_TS002_LoginFunctionality.java:57)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- r1dot1_TS002_TC003 -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="draftfail" time="0.0">
    <ignored/>
  </testcase> <!-- draftfail -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality" name="r1dot1_TS003_TC001" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot1_TS003_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality" name="r1dot2_TS006_TC001" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot2_TS006_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality" name="R1dot2_TS005_TC024" time="0.0">
    <ignored/>
  </testcase> <!-- R1dot2_TS005_TC024 -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="readExcelDraft" time="0.0">
    <ignored/>
  </testcase> <!-- readExcelDraft -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC005" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot1_TS001_TC005 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality" name="r1dot1_TS002_TC001" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot1_TS002_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality" name="r1dot2_TS005_TC001" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot2_TS005_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="loginPagefail" time="0.0">
    <ignored/>
  </testcase> <!-- loginPagefail -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC003" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot1_TS001_TC003 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality" name="r1dot1_TS003_TC002" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot1_TS003_TC002 -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="testHomepageFeaturedProduct" time="0.0">
    <ignored/>
  </testcase> <!-- testHomepageFeaturedProduct -->
  <testcase classname="com.tutorialNinja.functionalTests.Draft" name="testHomePageNav" time="0.0">
    <ignored/>
  </testcase> <!-- testHomePageNav -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality" name="r1dot1_TS001_TC001" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot1_TS001_TC001 -->
  <testcase classname="com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality" name="r1dot1_TS003_TC003" time="0.0">
    <ignored/>
  </testcase> <!-- r1dot1_TS003_TC003 -->
</testsuite> <!-- TutorialNinja functional Negative Test -->
